main.py

```diff
@@ -2,6 +2,33 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
 
 def main() -> QuerySet:
-    pass
+
+    genres = ["Western", "Action", "Dramma"]
+
+    actors = [
+        ("George Klooney", "Kianu Reaves", "Scarlett Keegan"),
+        ("Will Smith", "Jaden Smith", "Scarlett Johansson")
+    ]
+
+    for genre in genres:
+        Genre.objects.create(name=genre)
+
+    for last_name, first_name in actors:
```

Comment:
The tuple unpacking in the loop is incorrect. The `actors` list should contain tuples of two elements (first name and last name), but currently, it contains tuples of three elements.